client/client.go

```diff
@@ -22,57 +22,334 @@ limitations under the License.
 package client
 
 import (
+	"errors"
 	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
 
 	"github.com/fsouza/go-dockerclient"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/config"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/fs"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/network"
+	"github.com/intelsdi-x/snap-plugin-collector-docker/wrapper"
 	"github.com/opencontainers/runc/libcontainer/cgroups"
 )
 
-const endpoint string = "unix:///var/run/docker.sock"
+const (
+	endpoint         string = "unix:///var/run/docker.sock"
```

Comment:
I think that we should indicate in the documentation for this plugin, that it is assuming that docker in using default endpoint. Support for custom endpoints can be included into roadmap.